File: app/src/main/java/com/example/yujimomoi/a1daysummerintern/classFile/FieldArea.java
package com.example.yujimomoi.a1daysummerintern.classFile;

import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.util.Log;

/**
 * Created by yuji.momoi on 2016/07/07.
 */
public class FieldArea extends BaseObject{
	private LineData line_data;

	public FieldArea() {
		super(BaseObject.OBJ_TYPE_FIELD);
		LogPrint.getInstans().logWrite("create", "FieldArea", true);
		this.line_data = null;
	}

	public void init() {
		LogPrint.getInstans().logWrite("init", "FieldArea", true);
		this.line_data = new LineData();
		super.setObj(this);
	}

	public void draw(Canvas canvas) {
		this.line_data.draw(canvas);
	}

	public void setLineData(Paint lineColor, Point startPoint, Point endPoint) {
		this.line_data.setLineData(lineColor,startPoint, endPoint);
	}
}

File: app/src/main/java/com/example/yujimomoi/a1daysummerintern/classFile/FieldArea.kt
package com.example.yujimomoi.a1daysummerintern.classFile

/**
 * Created by yuji.momoi on 2016/07/07.
 */
class FieldArea {
    var fieldWidth: Int = 0
        private set
    var fieldHeight: Int = 0
        private set
    private val line_color: String?

    init {
        fieldWidth = 0
        fieldHeight = 0
        line_color = null
    }

    fun initField(width: Int, height: Int) {
        fieldWidth = width
        fieldHeight = height
    }

    fun initFielid() {
    }

    val fieldSise: FieldArea?
        get() = null

    fun getFieldSize() {
    }

    fun drawLine(point: Point) {
    }
}
